github/issue.go

```diff
@@ -111,15 +111,31 @@ func (g GitHub) maybeOpinion(issueHook *octokat.IssueHook) error {
 	return nil
 }
 
+func labelFromVersion(version, suffix string) string {
+	switch suffix {
+	// Dev suffix is associated with a master build.
+	case "dev":
+		return "version/master"
```

Comment:
There's no other way to learn about issues using a master build... If you exclude all `version/*` labels manually (which is cumbersome), you still have all the issues which are not providing a version number.